.github/workflows/macos.yml

```diff
@@ -0,0 +1,17 @@
+name: macOS
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  install:
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        compiler: [clang, gcc]
+
+    steps:
+      - name: Install latest Tig
+        run: CC=${{ matrix.compiler }} HOMEBREW_NO_AUTO_UPDATE=1 brew install --HEAD tig
```

Comment:
I don't think that Homebrew will honour the CC variable here. `HOMEBREW_CC=${{ matrix.compiler }}` may be a better choice.

Reply:
makes sense, done